spec/features/merchants/merchants_index_page_spec.rb

```diff
@@ -70,5 +70,47 @@
       click_link(@merchant_1.name)
       expect(current_path).to eq(admin_merchant_path(@merchant_1))
     end
+
+    it 'should enable/disable a merchant after corrisponding button has been clicked' do
+      admin = create(:admin)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+      visit merchants_path
+      within "#merchant-#{@merchant_1.id}" do
+        click_link 'Disable'
+        expect(current_path).to eq(merchants_path)
+        #link should now be to enalbe to the account
+        expect(page).to have_link("Enable")
+        @merchant_1.reload
+        expect(@merchant_1.enabled).to eq(false)
+      end
+      #flash message isnt occuring within css tag
+      expect(page).to have_content("#{@merchant_1.name} is now disabled")
+
+      within "#merchant-#{@merchant_2.id}" do
```

Comment:
What is line 90-99 testing that was not already tested in lines 79-88?

Reply:
Nothing really, just figured there wasn't really a reason to not test it